molecule/cookiecutter/scenario/driver/ec2/{{cookiecutter.molecule_directory}}/{{cookiecutter.scenario_name}}/create.yml

```diff
@@ -8,8 +8,9 @@
   vars:
     ssh_user: ubuntu
     ssh_port: 22
+    default_region: "{{ lookup('env', 'AWS_DEFAULT_REGION') | default('us-east-1', true) }}"
 
-    security_group_name: molecule
+    security_group_name_prefix: molecule
```

Comment:
Let's pick "molecule_prefix" and reuse this. I doubt we really need different prefixes on each resource type. By picking this we could make it a standard feature across multiple drivers.

@decentral1se what do you think?

Reply:
I want to be sure that different products that run tests in parallel use different security groups.
we can rename variables if needed.
the goal of the current name:
1) end-user should easily understand where this var will be used
2) during review in AWS, end-user should easily understand which test created resources